BaseFilter/sections/specific.txt

```diff
@@ -17182,8 +17182,8 @@ charismanews.com###articles > article.article:has(> div.contentBoxWrap a[href^="
 charismanews.com#$##headerAreaWrap { height: 180px !important; }
 !
 ! check-host.net
-check-host.net#?#a[href]:contains(/100[%]|[сcЗ]айт|[𝅺]/)
-check-host.net##div[id*="b"][id*="n"][id*="m"][id*="t"]
+check-host.net#?#div + *:contains(/100[%]|[сcЗ]айт|[𝅺]/)
```

Comment:
The rule removes a useful element [here](https://check-host.net/subnet-calculator).

Reply:
![image](https://github.com/AdguardTeam/AdguardFilters/assets/149155649/d4922bf0-3c23-4854-84e7-483d719702dc)
![image](https://github.com/AdguardTeam/AdguardFilters/assets/149155649/33822ca0-33d8-4267-bbd4-7b458e0746c4)

What am I missing? It looks the same to me.

Reply:
Never mind, I see what you mean; it only affects the Russian version of the site.